ndctl.spec.in

```diff
@@ -38,6 +38,9 @@ subsystem defines a kernel device model and control message interface for
 platform NVDIMM resources like those defined by the ACPI 6+ NFIT (NVDIMM
 Firmware Interface Table).
 
+%if 0%{?flatpak}
+%global _udevrulesdir %{_prefix}/lib/udev/rules.d
```

Comment:
Just curious about this part - do flatpak build roots not have this variable set up beforehand?

Reply:
Not at the moment, no. See https://src.fedoraproject.org/rpms/flatpak-rpm-macros/blob/rawhide/f/macros.flatpak

I'm planning to review it and create another PR with number of missing macros but likely I won't have time to look into this until this weekend.